Kernel/Storage/StorageManagement.cpp

```diff
@@ -219,7 +219,7 @@ UNMAP_AFTER_INIT void StorageManagement::determine_boot_device_with_partition_uu
     VERIFY(!m_storage_devices.is_empty());
     VERIFY(m_boot_argument.starts_with(partition_uuid_prefix));
 
-    auto partition_uuid = UUID(m_boot_argument.substring_view(partition_uuid_prefix.length()));
+    auto partition_uuid = UUID(m_boot_argument.substring_view(partition_uuid_prefix.length()), UUID::Endianness::Mixed);
```

Comment:
Hmm, is this correct? is the UUID mixed for all partition tables? or only GPT?

Reply:
Do you know about other partition tables that use UUIDs (GUIDs)? I don't so I think this is correct.

Reply:
If that's the only UUID partition table, sure :)